Home_task_14_1.py

```diff
@@ -0,0 +1,43 @@
+class Phone:
+    phone_number: str = ''
+    _count_calls: int = 0
+
+    def change_number(self, phone_number: str) -> str:
+        print(phone_number, ' has changed')
+        return phone_number
+
+    def output_count_callings(self) -> int:
```

Comment:
Намагайся робити назви релевантними.
От `change_number` - норм, хоча я б користувався стандартними `get_number/set_number`.
А цей доволі не зручний для розуміння операції.

```suggestion
    def get_calls_count(self) -> int:
```